packages/service-metrics/src/service-metrics.ts

```diff
@@ -4,14 +4,17 @@
 import { MeterProvider, PeriodicExportingMetricReader } from '@opentelemetry/sdk-metrics'
 import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-http'
 import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http'
-import { BasicTracerProvider, TraceIdRatioBasedSampler,
-         ParentBasedSampler, BatchSpanProcessor } from '@opentelemetry/sdk-trace-base'
```

Comment:
@nathanielc is it an accident you re-ordered these or was it to avoid a problem?